validator/cycle/process_tasks.py

```diff
@@ -62,6 +63,15 @@ async def _make_offer(node: Node, request: MinerTaskRequest, config: Config) ->
     )
 
 
+def _calculate_selection_probabilities(nodes: list[Node], random_weight: float = 0.005):
+    incentives = np.array([node.incentive for node in nodes])
```

Comment:
Don't really think we need numpy here but it will work i guess

Reply:
ded